pytorch_binding/README.md

```diff
@@ -23,15 +24,18 @@ installed. If you have a GPU, you should also make sure that
 `CUDA_HOME` is set to the home cuda directory (i.e. where
 `include/cuda.h` and `lib/libcudart.so` live). For example:
 
-```
-export CUDA_HOME="/usr/local/cuda"
-```
 
 Now install the bindings: (Please make sure the GCC version >= 4.9)
 ```
 cd pytorch_binding
 python setup.py install
 ```
+If installed successfully and you are using virtualenv, plz restart your venv
+```
+deactivate
+cd <your_python_virtualenv>/bin
+source activate
+```
```

Comment:
Out of curiosity, why would you need to restart the virtual env here?

Reply:
that is why, when I use python virtualenv,